packages/contract/src/DaimoAccount.sol

```diff
@@ -161,14 +181,36 @@ contract DaimoAccount is IAccount, UUPSUpgradeable, Initializable, IERC1271 {
     ) private view returns (bool) {
         if (signature.length < 1) return false;
 
-        // First bit identifies the keySlot
-        uint8 keySlot = uint8(signature[0]);
+        // First bit identifies the number of signatures
+        uint8 numSignatures = uint8(signature[0]);
+        if (numSignatures < signatureThreshold) return false;
+
+        // TODO: this requires all signatures be valid. What
+        // if 2 sigs are required, 3 get passed in, and 1 is
+        // invalid. Should op succeed?
```

Comment:
yeah wouldn't worry about it for now but probably for w/e we launch with we'd use a counter